class-01-responsive-mobile-first/pair-assignment/starter-code/styles/modules.css

```diff
@@ -66,3 +69,17 @@
          - The main-nav should be all on one row, with padding spaced better
            for a larger screen.
 */
+@media only screen and (min-width: 960px) {
```

Comment:
960px is a pretty high minimum for desktop mode. I'd recommend 600px - that's a pretty standard cutoff for desktop/mobile.